### PR TITLE
Added "screen" option for *NIX systems

### PR DESCRIPTION
**Pull request by zeryl**
Allows for generating a beehive that utilizes the *NIX program "screen", for easy review/watching of the server/workers

## Description
Added new option for --screen, to change the output of the batch file/shell script

## How Has This Been Tested?
Ran the location generator, and compared it's output, to my current use case.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by talentedmrripley** [comment]
Out of curiosity, why screen over dtach?

**Comment by zeryl** [comment]
@talentedmrripley only because it's what I use day to day.  Haven't seen dtach, but will look at it now.

**Comment by talentedmrripley** [comment]
Same functionality, just a bit lighter on resources.  
Here's an example:

dtach -n /tmp/home-pgm python /home/ripley/PokemonGo-Map/runserver.py -a ptc -u **** -p **** -st 15 -l "****, -****" -P 5000

**Comment by andrewda** [comment]
@talentedmrripley `screen` is much more commonly used than `dtach`.

**Comment by zeryl** [comment]
Missed a call to bash. Must have ran the wrong generated bash earlier. Fixed!  Re-pull if you have this.

File: Tools/Hex-Beehive-Generator/location_generator.py [hunk 1]
```diff
@@ -28,6 +26,14 @@
 st = args.steps
 wst = args.leaps
 
+if args.screen:
+    server_template = "screen -AmdS {screenname} bash -c \"python runserver.py -os -l '{lat} {lon}'\"\n"
```
Comment (Talamar1) [inline Tools/Hex-Beehive-Generator/location_generator.py]: You forgot your commas between {lat} and {lon}.

**Comment by AHAAAAAAA** [comment]
rebase

**Pull request closed without merging**